packages/table/src/Table/Table.stories.tsx

```diff
@@ -0,0 +1,602 @@
+import TableBody from '../TableBody/TableBody';
+import Row from '../Row/Row';
+import Cell from '../Cell/Cell';
+import useLeafygreenTable from '../useLeafygreenTable';
+import ExpandedContent from '../ExpandedContent/ExpandedContent';
+
+export default {
+  title: 'Components/TableNew',
+  component: Table,
+  argTypes: {
+    children: { control: 'none' },
+    darkMode: storybookArgTypes.darkMode,
+    ref: { control: 'none' },
+  },
+} as Meta<typeof Table>;
+
+const Template: ComponentStory<typeof Table> = args => {
+  const data = makeData(false, 100);
+  const columns = Object.keys(data[0]).filter(x => x !== 'renderExpandedContent' && x !== 'subRows')
+  return (
+    <Table {...args}>
+      <TableHead>
+        <HeaderRow>
+          {columns.map((columnName: any) => (
+            <HeaderCell key={columnName} columnName={columnName} />
+          ))}
+        </HeaderRow>
+      </TableHead>
+      <TableBody>
+        {data.map((row: any) => (
+          <Row>
+            {Object.keys(row).map((cellKey: string, index: number) => {
+              return <Cell key={`${cellKey}-${index}`}>{row[cellKey]}</Cell>;
+            })}
+          </Row>
+        ))}
+      </TableBody>
+    </Table>
+  );
+};
+
+export const Basic = Template.bind({});
+export const ZebraStripes = Template.bind({});
+ZebraStripes.args = {
+  shouldAlternateRowColor: true,
+};
+
+export const StickyHeaderRow = () => {
+  return <>TODO</>
+}
+
+export const NestedRows = () => {
+  const data = makeData(false, 100, 3);
+  const columns = Object.keys(data[0]).filter(x => x !== 'renderExpandedContent' && x !== 'subRows')
+  return (
+    <Table>
+      <TableHead>
+        <HeaderRow>
+          {columns.map((columnName: string) => (
+            <HeaderCell key={columnName} columnName={columnName} />
+          ))}
+        </HeaderRow>
+      </TableHead>
+      <TableBody>
+        {data.map((row: any) => (
+          <Row>
+            {columns.map((cellKey: string, index: number) => {
+              return <Cell key={`${cellKey}-${index}`}>{row[cellKey]}</Cell>;
+            })}
+            {row.subRows && row.subRows.map((subRow: any) => {
+              let subRowCellKeys = Object.keys(subRow).filter(x => x !== 'renderExpandedContent' && x !== 'subRows')
+              return (
+                <Row>
+                  {subRowCellKeys.map((cellKey: string) => (
+                    <Cell>
+                      {subRow[cellKey]}
+                    </Cell>
+                  ))}
+                </Row>
+              )
+            })}
+          </Row>
+        ))}
+      </TableBody>
+    </Table>
+  );
+};
+
+export const Sortable = () => {
+  return <>TODO</>
+}
+
+export const Selectable = () => {
+  return <>TODO</>
+}
+
+export const ExpandableContent = () => {
+  const data = makeData(false, 100, 3, 2);
+  const columns = Object.keys(data[0]).filter(x => x !== 'renderExpandedContent' && x !== 'subRows')
```

Comment:
What is the `renderExpandedContent` key doing in the data object? I'm not seeing it used anywhere. I think this is just bloat?